### PR TITLE
add kiwipy recipe

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

**Comment by ltalirz** [comment]
One of the tests hangs, unfortunately I cannot cancel all builds.
Package maintainer notified https://github.com/muhrin/kiwipy/issues/5

**Comment by mariusvniekerk** [comment]
Typically we don't run package tests here.  We test that things are packaged correctly (ie importable and correctly linked).  The actual tests are run upstream of us presumably.

File: recipes/kiwipy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - pika
+    - pyyaml
+
+
+test:
+  imports:
+    - kiwipy
+    - kiwipy.rmq
+  commands:
+    - python -m unittest discover
```
Comment (mariusvniekerk) [inline recipes/kiwipy/meta.yaml]: Remove this please

Comment (ltalirz) [reply recipes/kiwipy/meta.yaml]: Ok, done.
Sorry, I wasn't aware of this - I've always tried to include runs of (short) tests in my recipes so far and this is the first time someone has taken issue with it.

**Pull request merged**